python/src/CalibrationResult_doc.i.in

```diff
@@ -295,3 +295,13 @@ Returns
 -------
 grid : :class:`~openturns.GridLayout`
     Graph array."
+
```

Comment:
This change does not seem to be sufficient from my point of view. This is because the API doc prints :

![image](https://github.com/user-attachments/assets/c26b357c-89f1-4f5e-b92b-194a11907a47)

But `CalibrationResults` needs a `bayesian` boolean:

https://github.com/openturns/openturns/blob/a7e9d1ab1b4862d106f31c70f8c0ae115cababa4/lib/src/Uncertainty/Bayesian/openturns/CalibrationResult.hxx#L50

This prevents users from understanding that an extra boolean is needed. This is necessary when we design our own calibration method (say Approximate Bayesian Computation, see https://en.wikipedia.org/wiki/Approximate_Bayesian_computation), but want to use the post-processing provided by `CalibrationResults`.

Reply:
done